test/integration/error_spam_test.go

```diff
@@ -32,6 +32,8 @@ var stderrWhitelist = []string{
 	`kubectl`,
 	// slow docker warning
 	`slow|long time|Restarting the docker service may improve`,
+	// don't care if we can't push images to other profiles
+	`cache_images.go:.*error getting status`,
```

Comment:
I think if u change to warnning u wont need to whitelist anymore ?

Reply:
Correct. This is just defense in depth in case we change our mind about showing this message to users later.